docs/quickstart-cluster.rst

```diff
@@ -0,0 +1,262 @@
+.. _citus_quickstart:
+
+Citus Cluster Quick Start
+=========================
+
+In this guide we’ll create a Citus cluster with a coordinator node and two workers. Every node will have a secondary for failover. We’ll simulate failure in the coordinator and worker nodes and see how the system continues to function.
+
+For simplicity we’ll run all the nodes on a single machine, but in production the primary and secondary servers for each PostgreSQL node would run on independent machines.
+
+Before attempting this quickstart, you should try the :ref:`tutorial` for plain PostgreSQL nodes. The current guide relies on concepts explained there.
+
+Install pg_auto_failover and run a Monitor
+------------------------------------------
+
+pg_auto_failover requires a monitor node to keep track of global state and orchestrate actions on the other nodes. Follow the first two steps from the previous quickstart to get started:
+
+* :ref:`install`
+* :ref:`tutorial_run_monitor`
+
+.. note::
+
+  If you created a monitor *and registered nodes* in it as part of the previous quickstart guide, do not reuse the monitor. Create a new one with no nodes so that the cluster can be designated as type "citus" rather than type "postgres."
+
+Create a Citus Cluster
+----------------------
+
+pg_auto_failover can not only create nodes (as demonstrated in the previous quickstart), it can also configure them with the Citus extension. In this guide we'll use pg_auto_failover to create a coordinator node and two workers.
+
+First, install the Citus extension onto the system so pg_auto_failover can enable it on nodes:
+
+.. code-block:: bash
+
+  sudo yum install -y citus-rebalancer81_11
```

Comment:
I don't think the package has rebalancer in the name

Reply:
That dates from a very long time ago when we had a separate package for the rebalancer, with a dependency on the main Citus package. We might want to fix that indeed. Will have a look... but also, see below.

Reply:
The Citus tutorial now uses Docker and docker-compose and doesn't depend on RPM packages anymore, debian ones, and the community packages too.